src/unix/lwt_engine.ml

```diff
@@ -197,6 +197,57 @@ end
 
 class libev_deprecated = libev ()
 
+let unix_fd_to_fd (fd: Unix.file_descr) : int = Obj.magic fd
+
+class libuv () = object
+  inherit abstract
+
+  val loop = Luv.Loop.default ()
+
+  method loop = loop
+
+  method private cleanup = Luv.Loop.stop loop
```

Comment:
This method isn't called anywhere.

Reply:
This method is present in the libev code, but is it okay to remove?

Reply:
Ah yes, it's required by the base class. Sorry.